src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$day-size: 100px;
+$padding: 10px;
+$gap: 1px;
+$month-length: 31;
+$width: $day-size * 7 + $gap * 6 + $padding * 2;
+$week-days:
+  "tue" 101px,
```

Comment:
will be better to use date numbers here and calculate the margins value in cycle using some formula